crates/swc_ecma_minifier/tests/terser/compress/reduce_vars/conditional_nested_3/output.js

```diff
@@ -2,7 +2,7 @@ var n = 2, c = 0;
 (function f(a) {
     0 < n-- && g(a = 1);
     function g() {
-        a && c++;
```

Comment:
This seems dangerous

Reply:
It's fine if it's result of evaluating `0 < n--`, but it's wrong otherwise

Reply:
It is! I'm a bit shocked by this output at first, but it turns out fine.

Reply:
Oh sorry, this is wrong